### PR TITLE
Add DCAP and pallet_xcm benchmarks

### LINKED ISSUE #207: add benchmarked xcm weights
xcm weights were introduced in polkadot-v0.9.39, we are currently just using the test-weights: https://github.com/integritee-network/parachain/blob/b7c260abac8c607ed35123e539a6e7a9d8b6b2ce/polkadot-parachains/integritee-runtime/src/xcm_config.rs#L319

### PR DESCRIPTION
**Pull request by clangenb**
* Re-run the pallet-teerex benchmarks
* added the pallet-xcm benchmarks, closes #207.

### CODE REVIEW

**Review by OverOrion — APPROVED** [review]
LGTM!

File: polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+			.saturating_add(T::DbWeight::get().reads(1))
+	}
+	/// Storage: Benchmark Override (r:0 w:0)
+	/// Proof Skipped: Benchmark Override (max_values: None, max_size: None, mode: Measured)
+	fn execute() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 18_446_744_073_709_551 nanoseconds.
+		Weight::from_ref_time(18_446_744_073_709_551_000)
```
Comment (brenzi) [inline polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs:82]: this shouldn't have slipped our reviews. very funny execution time indeed. this most likely breaks receiving XCM messages and makes everything overweight. CC @clangenb

Comment (clangenb) [reply polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs:82]: Hmm, interesting. Funnily, moonbeam has the same weird weight for teleport assets, but their `execute()` weight seems fine:  https://github.com/moonbeam-foundation/moonbeam/blob/d4fdae0cf795ebb48b7a2a9c2167c7dae56e782f/runtime/common/src/weights/pallet_xcm.rs#L75

Comment (brenzi) [reply polkadot-parachains/integritee-runtime/src/weights/pallet_xcm.rs:82]: fixing this doesn't solve our problem, unfortunately. but needs fixing anyway

**Pull request merged**